test/e2e/scheduling/priorities.go

```diff
@@ -56,6 +58,17 @@ var podRequestedResource *v1.ResourceRequirements = &v1.ResourceRequirements{
 	},
 }
 
+var podwithLargeRequestedResource *v1.ResourceRequirements = &v1.ResourceRequirements{
```

Comment:
Looks like this var is used only in one of the tests. Please move it inside the test func.